### PR TITLE
User prompting, refresh upload, add to latest transaction

### PR DESCRIPTION
**Pull request by JohnathanZhou**
## Overview
This PR provides functionality for the MVP version of the app, which uploads a receipt and attaches it to the latest transaction the customer has. It also allows users to pull to refresh their transactions and rewards (next PR) screen. If there has been a new transaction added after the refresh, the user will be prompted to upload a receipt.

## Future
- [ ] (More immediate) Better checking of latest transaction and handling it in the state (RN it won't go away after you upload the picture) 
- [ ] Better handling for latest transactions, stretch is to just have each transaction have a button to click on that takes it to uploading a receipt

https://app.asana.com/0/1142438342256785/1148188905575529
https://app.asana.com/0/1142438342256785/1143295999106296

### CODE REVIEW

File: screens/Camera.js [hunk 1]
```diff
@@ -1,149 +1,174 @@
...
-                ]
+    .then(data => {
+      return data.json()
+    }).catch(err => {
+      console.error(err)
+    })
+    .then(data => {
+      let postUrl = data.secure_url
+      let currentDate = new Date()
+      return BASE('Receipts').create([
```
Comment (anniero98) [inline screens/Camera.js]: Nit: does this mean we can delete the `images` table now?

File: screens/Camera.js [hunk 2]
```diff
@@ -1,149 +1,174 @@
...
-          <View
-            style={{
-              flex: 1,
-              backgroundColor: 'transparent',
-              flexDirection: 'row'
-            }}>
-            <TouchableOpacity
+    } else {
+      return (
+        <View style={{ flex: 1 }}>
```
Comment (anniero98) [inline screens/Camera.js]: Just a reminder that we're all in `styled-components` now -- that can be in the next PR from part 2 tasks though

File: screens/HomeScreen.js [hunk 3]
```diff
@@ -1,45 +1,105 @@
...
+      redeemable: {},
+      announcements: '',
+      transactions: [],
+      latestTransaction: '',
+      refreshing: false, 
+      updates: false
     };
   }
 
+  // This is what runs when you pull to refresh. It is essentially the
```
Comment (anniero98) [inline screens/HomeScreen.js]: Curious if we're able to break part of it into a helper function?

File: screens/HomeScreen.js [hunk 4]
```diff
@@ -95,12 +149,12 @@ export default class HomeScreen extends React.Component {
...
                 styles.homeScreenFilename
               ]}>
-              <MonoText>{this.state.points} / 1000 </MonoText>
+              <MonoText>{this.state.points} total points </MonoText>
             </View>
             <Text style={styles.getStartedText}>
               {' '}
               {`${1000 -
-                parseInt(this.state.points)} points to your next reward`}
+                parseInt(this.state.points) % 1000} points to your next reward`}
```
Comment (anniero98) [inline screens/HomeScreen.js]: I added a column to Airtable called 'Unlocked Rewards' in the Customers table which does the calculation via formula!

File: screens/HomeScreen.js [hunk 5]
```diff
@@ -111,14 +165,43 @@ export default class HomeScreen extends React.Component {
...
         </ScrollView>
+        <View style={styles.tabBarInfoContainer}>
+          <Text style={styles.tabBarInfoText}>
+            Your transaction history:
+          </Text>
+          { this.state.transactions ? 
+            this.state.transactions.splice(0, 3).map(transaction => {
+              return(
+                <View
+                  key={transaction.get("transaction_id")}
```
Comment (anniero98) [inline screens/HomeScreen.js]: Nit: probably cleaner if we transform the transaction objects by mapping with a function as they come in from Airtable -- see #11 `createStoreData` etc in `storeHelpers.js`

File: screens/Camera.js (hunk 1 above)
Comment (JohnathanZhou) [reply screens/Camera.js]: yeah! images was just for testing -- receipts are where they are supposed to live

File: screens/Camera.js (hunk 2 above)
Comment (JohnathanZhou) [reply screens/Camera.js]: yes

File: screens/Camera.js (hunk 1 above)
Comment (anniero98) [reply screens/Camera.js]: Ok! It is gone now (^:

File: screens/HomeScreen.js (hunk 3 above)
Comment (JohnathanZhou) [reply screens/HomeScreen.js]: Yep

File: screens/HomeScreen.js (hunk 4 above)
Comment (JohnathanZhou) [reply screens/HomeScreen.js]: I think it's separate from this but I'll do some more handling in the next PR!

File: screens/HomeScreen.js (hunk 5 above)
Comment (JohnathanZhou) [reply screens/HomeScreen.js]: Done!

**Pull request merged**